examples/topics/geo/tile_xyzservices.py

```diff
@@ -1,3 +1,17 @@
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map.
+The xyzservices provides map tiles, which are typically used to display maps visualizations 
+and geographical location in web applications. This example demonstrates how to create a plot in Bokeh
+with a map background using tile_providers. The chart shows a map of a specific geographical region, 
+which is defined by the range bounds supplied in web mercator coordinates.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.Figure, bokeh.tile_providers.get_provider, bokeh.tile_provider.Vendors
```

Comment:
`Figure` (capital-F) was deprecated and removed at version 3.0 also the other APIs here are not actually used in this example
```suggestion
    :apis: bokeh.plotting.figure.add_tile
```

Reply:
All changes has been made